<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CdkForm</class>
 <widget class="QWidget" name="CdkForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一键输入cdk</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 0, 127)</string>
   </property>
   <property name="text">
    <string>cdk(神奇密码)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>一键输入cdk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
